curriculum/structure/superblocks/python-v9.json

```diff
@@ -160,6 +151,7 @@
       "comingSoon": true,
       "modules": [
         {
+          "comingSoon": true,
```

Comment:
How come these are becoming comingSoon? Consistency? Does it matter when the enclosing chapter is comingSoon?